src/pages/Hours.jsx

```diff
@@ -0,0 +1,268 @@
+import React, { useState } from 'react';
+import { Table, Input, Button, Typography, Space } from 'antd';
+import { SearchOutlined } from '@ant-design/icons';
+import { AFCBackend } from '../util/utils';
+import './Hours.css';
+
+const { Title } = Typography;
+
+const Hours = () => {
+  const [unapprovedVolunteersData, setUnapprovedVolunteersData] = useState([]);
+  const [selectedHours, setSelectedHours] = useState([]);
+  const [search, setSearch] = useState('');
+
+  const approveHours = async (userId, eventId) => {
+    try {
+      await AFCBackend.get(`/hours/approve/` + userId + '/' + eventId);
+      getUnapprovedVolunteers();
+    } catch (e) {
+      console.log(e.message);
+    }
+  };
+
+  const approveMultipleHours = async selectedHoursList => {
+    try {
+      for (let i = 0; i < selectedHoursList.length; i++) {
+        const userAndEventId = selectedHoursList[i].split(' ');
+        await AFCBackend.get(`/hours/approve/` + userAndEventId[0] + '/' + userAndEventId[1]);
+      }
+      getUnapprovedVolunteers();
+    } catch (e) {
+      console.log(e.message);
+    }
+  };
+
+  const declineHours = async (userId, eventId) => {
+    try {
+      await AFCBackend.get(`/hours/decline/` + userId + '/' + eventId);
+      getUnapprovedVolunteers();
```

Comment:
🚫 **[eslint]** <[no-use-before-define](https://eslint.org/docs/rules/no-use-before-define)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'getUnapprovedVolunteers' was used before it was defined.